packages/leancode_hooks/lib/src/use_bloc_state.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:bloc/bloc.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+
+/// Provides a [Cubit] or a [Bloc] current state. Forces [HookWidget] rebuild on
+/// state changed.
```

Comment:
```suggestion
/// Provides [Cubit]'s or [Bloc]'s current state. Forces [HookWidget] to
/// rebuild on state changed.
```

Reply:
what about "on state changed" -> "on state changes"/"on state change"?

Reply:
> on state changed

Right, missed that. Edited my suggestion